src/app/components/NavigationBar/NavigationBar.contract.ts

```diff
@@ -0,0 +1,28 @@
+import { Contracts } from "@arkecosystem/platform-sdk-profiles";
```

Comment:
Think it would be good to rename this to `<ComponentName>.contracts.tsx` (plural) to be consistent with other file names